transport-native-io_uring/src/main/java/io/netty/channel/uring/IOUringSubmissionQueue.java

```diff
@@ -86,7 +87,7 @@
         this.ringSize = ringSize;
         this.ringAddress = ringAddress;
         this.ringFd = ringFd;
-
+        this.submissionCallback = submissionCallback;
```

Comment:
Could we perhaps wire this up after construction, so we don't need to pass the callback through so many layers?

Reply:
@chrisvest I would like to keep it as it is ... wiring up things after construction is more error-prone and this one is easy to pass through.